test/common-utils/scenarios.json

```diff
@@ -41,6 +41,34 @@
             "common-utils": {}
         }
     },
+    "alma-8": {
```

Comment:
The tests are complaining about missing `alma-8.sh` file, which is added by https://github.com/devcontainers/features/pull/782

https://github.com/devcontainers/features/actions/runs/7674069177/job/20917976765?pr=811#step:4:7608

Can we either add the .sh file or remove the test scenario from this PR or to fix the test runs?

Reply:
Yeah I noticed. I just pushed the missing .sh files.

Reply:
Commit 1a425bd fixes the missing .sh test scripts.